src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * deprecated `KernelInterface::getName()` and the `kernel.name` parameter
+ * deprecated `kernel.root_dir` and `Kernel::getRootDir()`
```

Comment:
`KernelInterface::getRootDir()`

while at it, please put the param last as above :D